.github/workflows/integration-tests.yml

```diff
@@ -139,15 +139,15 @@ jobs:
         services:
             discovery-service:
                 image: ghcr.io/balhar-jakub/discovery-service:${{ github.run_id }}-${{ github.run_number }}
-            gateway-service:
+            zaas-service:
```

Comment:
Does it make sense? Is the reason to have 2 GW or 2 ZAAS? I mean it in general for the whole PR, because I rather assume to replace Gateway with SCGW (except the test of SCGW) and add one (or 2) instance of ZAAS